src/lib.rs

```diff
@@ -34,16 +34,14 @@ extern crate rusoto_core;
 extern crate rusoto_firehose;
 extern crate seahash;
 extern crate serde;
+#[macro_use]
```

Comment:
@pulltab looks like a result of removing `elastic_types_derive`, as you'll see later in the code the type of the buffer in ES is now simply a `Vec`